README.md

```diff
@@ -77,28 +79,50 @@ npm -g i firebase-tools
 on a mac, we recommend using [homebrew](https://brew.sh/)
 
 ```sh
-brew install java
+brew install openjdk
```

Comment:
Was this tested on macOS? I wouldn't recommend changing it if not.

Reply:
Cannot test it on mac. Need people on other OS to test the steps as well 😥

Reply:
As mentioned. The previous command java was incorrect. Should have always been `openjdk`